kolibri/plugins/coach/assets/src/views/common/status/HelpNeeded.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+
+  <div>
+    <component
+      :is="ratio ? LearnerProgressRatio : LearnerProgressCount"
+      :verb="VERBS.needHelp"
+      :icon="ICONS.help"
+      :total="total"
+      :count="count"
+      :verbosity="verbosity"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  import { VERBS, ICONS } from './constants';
+  import LearnerProgressCount from './LearnerProgressCount';
+  import LearnerProgressRatio from './LearnerProgressRatio';
+
+  export default {
+    name: 'HelpNeeded',
+    components: {
+      // eslint-disable-next-line vue/no-unused-components
+      LearnerProgressCount, // it is used, it's just referenced dynamically
+      // eslint-disable-next-line vue/no-unused-components
+      LearnerProgressRatio, // it is used, it's just referenced dynamically
```

Comment:
@jonboiser @rtibbles have either of you encountered this before? I'm surprised the linter is complaining

Reply:
its a bug. there's a patch on the eslint library to fix it.